### ISSUE TITLE
Wp-retina 2x plugin compatibility 

### ISSUE THREAD

**Issue by ferrbea** [sirulli/yoimages#31]
Provide support for Wp-retina 2x plugin as suggested by user wp-fred.

**Comment by fagia** [comment]
Pushed to dedicated branch: [retina_cropping](https://github.com/sirulli/yoimages/tree/retina_cropping)
Please evaluate, test and translate it.

Install the [WP Retina 2x plugin](https://wordpress.org/plugins/wp-retina-2x/ "") and then go to your test Wordpress plugins directory via terminal and there:
```
cd yoimages
git checkout retina_cropping
php composer.phar update
```

**Comment by fagia** [comment]
released with version 0.0.3
